Resource.rdef

```diff
@@ -0,0 +1,1160 @@
+
+resource app_flags B_SINGLE_LAUNCH;
+
+resource app_flags B_SINGLE_LAUNCH;
```

Comment:
looks like line 4 through 19 can be removed, as they are duplicated and outdated.